src/openprocurement/api/tests/base.py

```diff
@@ -15,8 +15,15 @@
 
 
 def snitch(func):
+    """
+        snitch method get test without test_ part in title
```

Comment:
This method is used to add test function to TestCase classes.
snitch method gets test function and returns a copy of this function 
with 'test_' prefix at the beginning (to identify this function as 
an executable test).
It provides a way to implement a storage (python module that 
contains non-executable test functions) for tests and to include 
different set of functions into different test cases.